### PR TITLE
ES add format to the query string

### PR DESCRIPTION
**Pull request by MosheEichler**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [link to the issue](https://jira-hq.paloaltonetworks.local/browse/XSUP-27285)

## Description
In order to support all datetime formats we add the format key to the query search
see [here](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-range-query.html#range-query-field-params)

### CODE REVIEW

File: Packs/Elasticsearch/ReleaseNotes/1_3_16.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### Elasticsearch v2
+- Fixed an issue where **fetch incidents** stopped fetching events due to a wrong datetime format.
```
Comment (ShirleyDenkberg) [inline Packs/Elasticsearch/ReleaseNotes/1_3_16.md:4]:
```suggestion
- Fixed an issue where **fetch incidents** stopped fetching events due to an incorrect datetime format.
```

**Comment by ShirleyDenkberg** [comment]
@tkatzir @bziser @DeanArbel Doc review completed.

File: Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py [hunk 2]
```diff
@@ -304,9 +265,7 @@ def search_command(proxies):
     time_range_dict = None
     if timestamp_range_end or timestamp_range_start:
         time_range_dict = get_time_range(time_range_start=timestamp_range_start, time_range_end=timestamp_range_end,
-                                         time_field=timestamp_field,
-                                         datetime_format=get_datetime_field_format(es, index, timestamp_field),
-                                         )
+                                         time_field=timestamp_field)
```
Comment (tkatzir) [inline Packs/Elasticsearch/Integrations/Elasticsearch_v2/Elasticsearch_v2.py:268]:
```suggestion
                                         time_field=timestamp_field,
                                         )
```

**Review by tkatzir — APPROVED** [review]
Good!
Approved, with one minor comment.

**Comment by bziser** [comment]
Looks good
please fix the pre-commit to pass and add link to the bug

**Review by bziser — APPROVED** [review]

**Pull request merged**